### PR TITLE
feat: add funding reason to bulk upload create field reference method

### PR DESCRIPTION
**Pull request by CaiWillis**
TIS21-5477: Bulk upload - Create post - add new field

### CODE REVIEW

File: reference-client/pom.xml [hunk 1]
```diff
@@ -65,6 +65,12 @@
       <artifactId>springfox-swagger2</artifactId>
       <version>2.8.0</version>
     </dependency>
+    <dependency>
```
Comment (CaiWillis) [inline reference-client/pom.xml:68]: This was added automatically by intellij, will remove

**Comment by CaiWillis** [comment]
image
Guessing the issue with the test is to do with the equivalence test for the ParameterizedTypeReference.

**Comment by CaiWillis** [comment]
Disagree with the sonar analysis code smells

**Comment by CaiWillis** [comment]
> > Disagree with the sonar analysis code smells
> 
> It seems that it's okay to remove `eq()` in the verify block.

I get the following if I remove any of the eq()

image

**Review by hihilary — APPROVED** [review]

**Pull request merged**